dev/release/00-prepare.sh

```diff
@@ -61,6 +53,27 @@ update_versions() {
   git add configure.ac meson.build
   cd -
 
+  cd "${SOURCE_DIR}/../../ci"
+  sed -i.bak -E -e \
+    "s/^pkgver=.+/pkgver=${r_version}/" \
+    PKGBUILD
+  rm -f PKGBUILD.bak
+  git add PKGBUILD
+  sed -i.bak -E -e \
+    "s/arrow-[0-9.]+[0-9]+/arrow-${r_version}/g" \
```

Comment:
Could you use `${version}` instead of `${r_version}`? We don't need to use `${r_version}` here.

Reply:
I used `r_version` there for convenience of regular expression matching. It doesn't really matter which you use because either way the "development" URL it creates is invalid; it's only there so that the release version gets set correctly by the script.

Reply:
I know that the URL is invalid.
`${r_version}` is the special version just for R. I don't want to use `${r_version}` for not R related code. If we use `${r_version}` here, people who may think that this is R related code or why do we need to use `${r_version}` here instead of general `${version}`.